components/cards/card.js

```diff
@@ -0,0 +1,12 @@
+const addToCartBtn = document.querySelector(".card-addtocart");
+const card = document.querySelector(".card-verticle");
+
+card.addEventListener("mouseover", () => {
+  addToCartBtn.style.opacity = 1;
+});
```

Comment:
You can use `:hover` pseudo class for this. why Javascript for hover effects? Any specific reason?

Reply:
Nothing specific, I thought it could be achieved only by using Js.